content/30.conclusions.md

```diff
@@ -8,14 +8,14 @@ Group D honoree counts are influenced from Spain as well as Latin America.
 In such cases, our analyses may substantially understate the extent to which minoritized scientists are underrepresented among honorees and authors.
 
 Biases in authorship practices may also result in our underestimation of the composition of minoritized scientists within the field.
-We estimate the composition of the field using corresponding author status, but in neuroscience [@doi:10.1101/275362] and other disciplines [@doi:10.1371/journal.pbio.2004956] women are underrepresented among such authors.
+We estimated the composition of the field using corresponding author status, but in neuroscience [@doi:10.1101/275362] and other disciplines [@doi:10.1371/journal.pbio.2004956] women are underrepresented among such authors.
 Such an effect would cause us to underestimate the number of women in the field.
 Though this effect has been studied with respect to gender, we are not aware of similar work examining race, ethnicity, or name origins.
 
+We acknowledged that our supervised learning approaches are neither error-free nor bias-free.
```

Comment:
I'm not an expert at hyphenation rules. Is this the right hyphenation?

Reply:
Good question! I just looked it up. From [The Chicago Manual of Style](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Compounds/faq0021.html):
> A compound formed with free as the second element is hyphenated both before and after the noun it refers to. (Whether such a phrase follows a verb is irrelevant.) Please see CMOS 7.89, section 3, under the word free:
>
> toll-free number
accident-free driver
the number is toll-free
the driver is accident-free

From [UW Madison Editorial Style Guide](https://editorial-styleguide.umark.wisc.edu/term/hyphens/):
> Hyphenate compound modifiers preceding a noun: well-run establishment, ill-fitting garment, full-time job, smoke-free restaurant. A compound modifier following the noun it describes does not require a hyphen, but it is not incorrect to use one: The restaurant is smoke free. 

If we can remove the hyphens, I think we should remove them.